getting-started/authenticating-users/app.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+    "fmt"
+    "log"
+    "net/http"
+    "io/ioutil"
+    "os"
+    "encoding/json"
+    "github.com/dgrijalva/jwt-go"
+)
+
+var AUDIENCE string
+var CERTIFICATES = make(map[string]string)
+
+
+func main() {
+    http.HandleFunc("/", indexHandler)
+
+    port := os.Getenv("PORT")
+    if port == "" {
+        port = "8080"
+        log.Printf("Defaulting to port %s", port)
+    }
+
+    log.Printf("Listening on port %s", port)
+    log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+}
+
+func certs() map[string]string {
+    const url = "https://www.gstatic.com/iap/verify/public_key"
+
+    if len(CERTIFICATES) == 0 {
+        resp, err := http.Get(url)
+        if err != nil {
+            log.Printf("Failed to fetch certificates: %s", err)
+            return CERTIFICATES
+        }
+
+        defer resp.Body.Close()
+        body, err := ioutil.ReadAll(resp.Body)
+        if err != nil {
+            log.Printf("Error reading certs: %s", err)
+            return CERTIFICATES
+        }
+
+        err = json.Unmarshal(body, &CERTIFICATES)
```

Comment:
Less important, but generally prefer inline error declaration in if statement
(see https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors)

Reply:
Since this is a sample program, not a library, the documentation decided that the user should just see "Hello None" in case of errors. So errors are handled by logging them for the developer if needed, and just returning that string.